components/grid-info.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="block">
+    <div class="grid">
+      <div class="block">
+        <div class="grid__block">
+          <div class="title">Open Source</div>
+          <div class="text">Исходный код открыт для исследования и обслуживания</div>
```

Comment:
it should has 2 lines

Now


![image](https://github.com/user-attachments/assets/0225e97d-61b5-4336-bfe2-54a1262d48d1)

Should be 


![image](https://github.com/user-attachments/assets/0317d3a4-4db1-44a4-b84e-5d690c2064b7)